secure/README.md

```diff
@@ -137,6 +137,13 @@ Using a self-signed certificate in a production system is strongly discouraged.
         -file <CA_REPLY_CRT>
     ```
 
+## Kerberos in High Availability mode
+
+In High Availability (HA) mode, a Dataproc cluster will have 3 masters. Such clusters, when Kerberized through this script, will have 3 KDCs, one on each of the Dataproc masters.
```

Comment:
Nit: please add a reference for a 3-KDC setup

Reply:
Added a link to the MIT doc.